packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md

```diff
@@ -3,7 +3,9 @@
    To handle empty `companyId` values, you can define the ACS URL as:
 
    ```bash
-   ${empty org.companyId ? 'https://login.myapp.io' : 'https://login.myapp.io?connection=' + org.companyId}
+   ${empty org.companyId ? 'login.myapp.io' : 'login.myapp.io?connection=' += org.companyId}
    ```
 
-   This expression handles both scenarios where `companyId` is populated or empty.
\ No newline at end of file
+   This expression handles both scenarios where `companyId` is populated or empty.
+
+   > **Note:** Only HTTPS URLs are allowed for ACS URLs. You don't need to add `https://` to the URL string.
```

Comment:
Wow, didn't know this! Did you get this from experience or elsewhere in our docs? I've typically been including `https://` in my ACS URLs and I had a vague memory that we required it but don't have an environment up right now to check.

Reply:
Sorry @jackward-okta , my bad: I misunderstood Abbas's comment about this field. I'll remove this note.